src/inventory_client/inventory_client.go

```diff
@@ -350,9 +350,9 @@ func (c *inventoryClient) ClusterLogProgressReport(ctx context.Context, clusterI
 	}
 }
 
-func (c *inventoryClient) HostLogProgressReport(ctx context.Context, clusterId string, hostId string, progress models.LogsState) {
+func (c *inventoryClient) HostLogProgressReport(ctx context.Context, infraEnvId string, hostId string, progress models.LogsState) {
 	_, err := c.ai.Installer.UpdateHostLogsProgress(ctx, &installer.UpdateHostLogsProgressParams{
-		ClusterID: strfmt.UUID(clusterId),
+		ClusterID: strfmt.UUID(infraEnvId),
```

Comment:
Why the name of the parameter is still ClusterID?
Don't we need an API with InfraEnvID for a case they are different?

Reply:
fixed. reverted to previous implementation. I am gonna change it in a different ticket